### PR TITLE
RDI: adding write-behind configuration section

### PR DESCRIPTION
**Pull request by redima**
As discussed with @yaronp68:
* Added a new section for write-behind
* two documents:
- * `configuration-guide.md` - configuration details for various datastores
- *  `recepies.md` - placeholder for wb recipes (from @loriotpiroloriol )

### CODE REVIEW

File: content/rdi/write-behind/configuration-guide.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+categories: ["redis-di"]
+aliases: 
+draft: 
+hidden: false
+---
+
+This guide provides details about configuring write-behind target connections.
+
+## Overview
+Write-behind target connections are used in RDI write-behind jobs (commonly referred as recipes) to replicate changes to when they are captured in a RDI-enabled Redis Enterprise database. The connections must be configured in the `config.yaml` before deploying any jobs and must follow one of the formats shown below. Multiple connections can be specified in the `connections` section at the same time.
```
[the remark this answers is not in the document]
Comment (nermiller) [reply content/rdi/write-behind/configuration-guide.md:16]: @redima I need a little clarification about this sentence: "Write-behind target connections are used in RDI write-behind jobs (commonly referred as recipes) to replicate changes to when they are captured in a RDI-enabled Redis Enterprise database." It's unclear to me what "to when they are captured" means.

File: content/rdi/write-behind/configuration-guide.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        database: rdi_wb_database
+        user: MYDOMAIN\rdi_service_account  # company-domain\service-account
+        password: secret                    # NTLM requires to provide a password
+```
+
+> Note: User must be specified with the domain name for Windows Authentication to work
+
+**For Kerberos:**
+
+* Please contact Redis team
```
Comment (nermiller) [inline content/rdi/write-behind/configuration-guide.md:87]: @redima Consider adding the contact information

File: content/rdi/write-behind/configuration-guide.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+After configuring the RDI connection and deploying the write-behind job, the operator can check whether RDI is using the expected authentication mechanism and protocol by running the following SQL query (note, this may require `sysadmin` role):
```
Comment (nermiller) [inline content/rdi/write-behind/configuration-guide.md:89]:
```suggestion
After you configure the RDI connection and deploy the write-behind job, run the following SQL query to have the operator check if RDI is using the expected authentication mechanism and protocol. **Note:** This operation may require `sysadmin` role.
```

File: content/rdi/write-behind/configuration-guide.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+```sql
+SELECT session_id, auth_scheme FROM sys.dm_exec_connections;
+```
+
+The results will indicate which `auth_scheme` is used by each session and may take the following values: `SQL`, `NTLM` and `Kerberos`.
```
Comment (nermiller) [inline content/rdi/write-behind/configuration-guide.md:95]:
```suggestion
The results indicate which `auth_scheme` is used by each session and may take values `SQL`, `NTLM`, and `Kerberos`.
```

File: content/rdi/write-behind/configuration-guide.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+### Azure Active Directory
+
+* Please contact Redis team
```
Comment (nermiller) [inline content/rdi/write-behind/configuration-guide.md:99]: @redima consider adding contact info

**Review by nermiller — COMMENTED** [review]
@redima some feedback for you. Thank you for expanding the content!

File: content/rdi/write-behind/configuration-guide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (redima) [reply content/rdi/write-behind/configuration-guide.md:16]: sure, perhaps need a comma to read it like below:
> Write-behind target connections are used in RDI write-behind jobs (commonly referred as recipes) to replicate changes to, when they are captured in RDI...

[the remark this answers is not in the document]
Comment (redima) [reply content/rdi/write-behind/configuration-guide.md:16]: or maybe:
> ... after they are captured in RDI

File: content/rdi/write-behind/configuration-guide.md (hunk 2 above)
Comment (redima) [reply content/rdi/write-behind/configuration-guide.md:87]: @yaronp68 please take a look - what do we want to put in here?

File: content/rdi/write-behind/configuration-guide.md (hunk 5 above)
Comment (redima) [reply content/rdi/write-behind/configuration-guide.md:99]: same as above. let's see what Yaron will reply with.

Comment (yaronp68) [reply content/rdi/write-behind/configuration-guide.md:99]: @redima please remove the kerberos & Azure AD section (or say not supported for now)

File: content/rdi/write-behind/configuration-guide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (elena-kolevska) [reply content/rdi/write-behind/configuration-guide.md:16]:
```suggestion
Write-behind target connections are the connections established between the RDI instance and the target database in a [write-behind scenario]({{</* relref  "/rdi/quickstart/write-behind-guide.md" */>}})`. They replicate the changes captured in an RDI-enabled Redis Enterprise database to the target database. 
The connections must be configured in the `config.yaml` before deploying any jobs and must follow one of the formats shown below. Multiple connections can be specified in the `connections` section at the same time.
```

@redima

[the remark this answers is not in the document]
Comment (elena-kolevska) [reply content/rdi/write-behind/configuration-guide.md:16]: @redima  It didn't feel like the connection configuration section is the best one to introduce the recipes. I think they might be a better fit for the quickstart section, and we link to it from here. What do you think?

**Comment by redima** [comment]
@yaronp68 @elena-kolevska is there anything else needed for this PR? Please approve if looking good to you. Thanks.

**Comment by elena-kolevska** [comment]
Hey @redima . I made a few small suggestions, please review and let's get this merged in!

**Comment by redima** [comment]
> Hey @redima . I made a few small suggestions, please review and let's get this merged in!

Hi @elena-kolevska I'm not able to see anything :( Can you please ping me tomorrow so we can take a look? Thanks

**Pull request merged**